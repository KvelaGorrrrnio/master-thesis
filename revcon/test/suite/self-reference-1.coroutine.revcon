@coroutine

coroutine main(int a) {
  a += 5
  a += a
}
