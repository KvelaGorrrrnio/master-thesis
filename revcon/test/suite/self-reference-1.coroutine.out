The variable a has been self-referenced in the statement: 'a += a'
